lib/jsdom/browser/default-stylesheet.js

```diff
@@ -437,95 +400,15 @@ input {
     cursor: auto;
 }
 
-input[type="search" i] {
+input[type="search"] {
     -webkit-appearance: searchfield;
     box-sizing: border-box;
 }
 
-input::-webkit-textfield-decoration-container {
-    display: flex;
-    align-items: center;
-    -webkit-user-modify: read-only !important;
-    content: none !important;
-}
-
-input[type="search" i]::-webkit-textfield-decoration-container {
-    direction: ltr;
-}
-
-input::-webkit-clear-button {
-    -webkit-appearance: searchfield-cancel-button;
-    display: inline-block;
-    flex: none;
-    -webkit-user-modify: read-only !important;
-    -webkit-margin-start: 2px;
-    opacity: 0;
-    pointer-events: none;
-}
-
-input:enabled:read-write:-webkit-any(:focus,:hover)::-webkit-clear-button {
-    opacity: 1;
-    pointer-events: auto;
-}
-
-input[type="search" i]::-webkit-search-cancel-button {
-    -webkit-appearance: searchfield-cancel-button;
-    display: block;
-    flex: none;
-    -webkit-user-modify: read-only !important;
-    -webkit-margin-start: 1px;
-    opacity: 0;
-    pointer-events: none;
-}
-
-input[type="search" i]:enabled:read-write:-webkit-any(:focus,:hover)::-webkit-search-cancel-button {
-    opacity: 1;
-    pointer-events: auto;
-}
-
-input[type="search" i]::-webkit-search-decoration {
-    -webkit-appearance: searchfield-decoration;
-    display: block;
-    flex: none;
-    -webkit-user-modify: read-only !important;
-    -webkit-align-self: flex-start;
-    margin: auto 0;
-}
-
-input[type="search" i]::-webkit-search-results-decoration {
-    -webkit-appearance: searchfield-results-decoration;
-    display: block;
-    flex: none;
-    -webkit-user-modify: read-only !important;
-    -webkit-align-self: flex-start;
-    margin: auto 0;
-}
-
-input::-webkit-inner-spin-button {
-    -webkit-appearance: inner-spin-button;
-    display: inline-block;
-    cursor: default;
-    flex: none;
-    align-self: stretch;
-    -webkit-user-select: none;
-    -webkit-user-modify: read-only !important;
-    opacity: 0;
-    pointer-events: none;
-}
-
-input:enabled:read-write:-webkit-any(:focus,:hover)::-webkit-inner-spin-button {
-    opacity: 1;
-    pointer-events: auto;
-}
-
 keygen, select {
```

Comment:
keygen was removed from the spec in January: https://gist.github.com/domenic/ec8b0fc8ab45f39403dd